.github/workflows/spelling.yml

```diff
@@ -2,14 +2,21 @@ name: Spell checking
 on:
   pull_request_target:
   push:
+  issue_comment:
 
 jobs:
   build:
     name: Spell checking
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.0.0
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
       with:
-        ref: ${{ github.event.pull_request.head.sha }}
-        fetch-depth: 5
-    - uses: check-spelling/check-spelling@0.0.17-alpha
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
```

Comment:
This switches to using a `merge` head for PRs.

`fetch-depth` is no longer necessary, as the bot will flag all instances of words in all lines whether they were added in recent commits or not. (The bot is now significantly faster, and flags all words in the first pass, and does reporting based on a second pass of identified words.)